src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+        for (int e = 0; e < figures.length / 2; e++) {
```

Comment:
```suggestion
        for (int i = 0; i < figures.length / 2; i++) {
```
you can use i here as well, it has different scope with i in the loop above